src/App/root_reducer.js

```diff
@@ -0,0 +1,8 @@
+import {combineReducers} from 'redux';
+import login from '../Login/reducer';
```

Comment:
we can probably alias a root folder to `src`, so instead, we write code like `import login from '../Login/reducer'`, we write `import login from 'Login/reducer'`